app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -189,16 +189,22 @@ private void showAppView(AppViewModel appViewModel) {
 
   private void sendAppViewLoadAnalytics(AppViewModel appViewModel) {
     AppModel appModel = appViewModel.getAppModel();
+    String developerName = "";
```

Comment:
This code isn't actually being used by us for anything since we're not using Aptoide's analytics tracking, but it is getting called & I was running into some null pointer errors with these fields. I could have just commented out the code, but adding the null checks was just as easy.